src/print.rs

```diff
@@ -19,6 +19,7 @@ const PROMPT_ORDER: &[&str] = &[
     "python",
     "go",
     "cmd_duration",
+    "jobs",
```

Comment:
Much like in spaceship, I think this would be better placed after `line_break`, so it would appear beside the character.

Reply:
@matchai I've made this change, but in my opinion, it looks kind of weird:

![Shot 2019-08-12 at 08 47 50](https://user-images.githubusercontent.com/30328854/62851437-fa9e8d80-bcdd-11e9-9fc5-2cb17941e540.jpg)

Reply:
Here's what the implementation from spaceship looks like, which I think is a little closer to what I had in mind.

![image](https://user-images.githubusercontent.com/4658208/62881048-d605de00-bcfc-11e9-89e6-bf80ef35183d.png)

Perhaps we can strip away the spacing after the `✦` and the prefix of the module.

Reply:
Understand @matchai ... I like that a lot better ... I have updated the code